sdk/servicebus/service-bus/src/core/batchingReceiver.ts

```diff
@@ -359,7 +353,7 @@ export class BatchingReceiver extends MessageReceiver {
         }
       };
 
-      const addCreditAndSetTimer = (reuse?: boolean) => {
+      const addCreditAndSetTimer = (reuse?: boolean): void => {
```

Comment:
@ramya-rao-a  what would be the appropriate type for `addCreditAndSetTimer`  ?

Reply:
void is right